common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -4,24 +4,80 @@
  */
 package com.wynntils.core.features.overlays;
 
-// TODO: This class is very basic atm
 public class OverlayPosition {
-    // Top-Left point of Overlay
-    private int x;
-    private int y;
 
-    private OverlayPosition() {}
+    private int verticalOffset;
+    private int horizontalOffset;
 
-    public OverlayPosition(int x, int y) {
-        this.x = x;
-        this.y = y;
+    private VerticalAlignment verticalAlignment;
+    private HorizontalAlignment horizontalAlignment;
+
+    private AnchorNinth anchorNinth;
+
+    public OverlayPosition(
+            int verticalOffset,
+            int horizontalOffset,
+            VerticalAlignment verticalAlignment,
+            HorizontalAlignment horizontalAlignment,
+            AnchorNinth anchorNinth) {
+        this.verticalAlignment = verticalAlignment;
```

Comment:
This alignment is *relative* to the "ninth" sector? Otherwise it seems redundant.

Reply:
What do you mean? Alignments are used to position the overlay in it's ninth with the specified offsets too.

Reply:
So what I mean is that the ninths/sectors themselves are aligned relative to the entire screen. So the "first" is TopLeft, which mean it's aligned towards top and left on the screen. Then the "Alignment" here is alignment is alignment within the sector, so you can have something relative to the "top left" of the TopLeft ninth. One one hand, it's clear as day, and you could not really do it differently, but on the other hand, the word "alignment" becomes a bit fuzzy, since you need to be super clear about *what* you are aligning against.

I was mostly thinking out loud if there could be a way to make this completely unambiguous. But maybe the current code is the best we can do.

Reply:
I think we need to have a way to align to overlays in their ninths any way, and we even will use negative alignment offsets. Setting these values will become the job of the overlay management GUI, which should save overlay position in the most optimal way.